content/de/docs/glossary.md

```diff
@@ -0,0 +1,179 @@
+
+{{% def id="LE" name="Let's Encrypt" abbr="LE" %}} Die [Zertifizierungssstelle](#def-CA) betrieben von [ISRG](#def-ISRG). [Wikipedia](https://en.wikipedia.org/wiki/Let%27s_Encrypt) {{% /def %}}
+
+{{% def id="mixed-content" name="Mixed content" %}} Wenn eine HTTPS Webseite Unterresourcen (Javascript, CSS oder Bilder) über HTTP lädt. [Browser](#def-web-browser) blockieren möglicherweise gemischten Inhalt oder markieren die Seite als weniger sicher, wenn gemischter Inhalt vorhanden ist: https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content. Um das Problem zu lösen, muss ein Webentwickler die Seite so ändern, dass alle Ressourcen HTTPS URLs benutzen. [Entwicklerwerkzeuge](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools), eingebaut in Browser, können benutzt werden, um herauszufinden, welche Resourcen Probleme mit gemischten Inhalten verursachen. {{% /def %}}
+
+{{% def id="OCSP" name="Online Certificate Status Protocol" abbr="OCSP" abbr_first="1" %}} Eine Methode zur Überprüfung des [Sperrstatus](#def-revocation) eines [Zertifikats](#def-leaf). In anderen Worte, ein Weg zu überprüfen, ob eine [Certificate Authority](#def-CA) anzeigt, dass das Zertifikat nicht länger gültig sein sollte, auch wenn das Ablaufdatum noch nicht erreicht ist. Diese Anfragen können Probleme mit der Privatsphäre verursachen, denn es erlaubt der Zertifizierungsstelle und Internet Service Providern direkte Beobachtung, wer welche Seiten besucht hat. [Wikipedia](https://en.wikipedia.org/wiki/Online_Certificate_Status_Protocol) {{% /def %}}
+
+{{% def id="OCSP-must-staple" name="OCSP Must-Staple" %}} Eine [Zertifikat](#def-leaf) Erweiterung, die den [Browser](#def-web-browser) informiert, dass der [Web Server](#def-web-server) mit dem Zertifikat [OCSP Stapling](#def-OCSP-staping) benutzen muss. Es wird benutzt, um einen aktuellen [Sperrstatus](#def-revocation) des [Zertifikats](#def-leaf) vom Webserver bei jeder Verbindung bestätigt zu bekommen, was Sperrungen mehr sinnvoll macht. [Let's Encrypt](#def-LE) kann Zertifikate mit OCSP Must-Staple [Erweiterung](#def-extension) ausstellen. [Mozilla Security Blog](https://blog.mozilla.org/security/2015/11/23/improving-revocation-ocsp-must-staple-and-short-lived-certificates/) [RFC 7633](https://tools.ietf.org/html/rfc7633) {{% /def %}}
+
+{{% def id="OCSP-stapling" name="OCSP stapling" %}} Ein Weg für [Web Server](#def-web-server), um zum [Browser](#def-web-browser) eine [OCSP](#def-OCSP) Antwort signiert von der [Zertifizierungsstelle](#def-CA) zu senden, sodass der Browser nicht selbst eine zweite OCSP Anfrage zur CA senden muss, verbessert Geschwindigkeit und Privatsphäre. Auch bekannt als TLS Certificate Status Request extension. [Wikipedia](https://en.wikipedia.org/wiki/OCSP_stapling) [Cloudflare](https://blog.cloudflare.com/high-reliability-ocsp-stapling/) {{% /def %}}
```

Comment:
```suggestion
{{% def id="OCSP-stapling" name="OCSP stapling" %}} Ein Weg für [Web Server](#def-web-server), um zum [Browser](#def-web-browser) eine [OCSP](#def-OCSP) Antwort signiert von der [Zertifizierungsstelle](#def-CA) zu senden, sodass der Browser nicht selbst eine zweite OCSP Anfrage zur CA senden muss, verbessert Geschwindigkeit und Privatsphäre. Auch bekannt als TLS Certificate Status Request extension. [Wikipedia](https://de.wikipedia.org/wiki/Online_Certificate_Status_Protocol_stapling) - [Cloudflare](https://blog.cloudflare.com/high-reliability-ocsp-stapling/) {{% /def %}}
```
Wikipedia de, and "-" (Yes, it was a bug in the English version: https://github.com/letsencrypt/website/compare/master...tdelmas:glossary_fr#diff-89128f046390cc560097e9389addba36)